### DISCUSSION TITLE
better error page for substituted domains

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1983] [Ideas]
currently, if a user opens any google page from the gui (for example, when pressing f1 for help), he will get to the "*** is blocked" error page. 

first, the user may be disoriented by the error. second, it is just inconvenient to enter the correct address manually

the solution is a special error page for substituted domains with an option to go to the original ones. like this:

![error](https://user-images.githubusercontent.com/72306717/171650476-61d69770-7ae5-4e81-99cb-08db93da5e05.png)

**Comment by networkException** [comment]
Good idea!

I think think we will want or be able to include the clear name / proceed link though
